lib/screens/login_screen.dart

```diff
@@ -92,63 +88,93 @@ class _LoginScreenState extends State<LoginScreen>
+                    style: const TextStyle(
+                      height: 2,
+                      color: Colors.black,
                     ),
-                    const SizedBox(height: 10),
-                    SizedBox(
-                      width: 200,
-                      child: TextField(
-                        controller: _passwordController,
-                        obscureText: true,
-                        decoration: const InputDecoration(
-                          border: OutlineInputBorder(),
-                          labelText: 'Password',
+                    children: <TextSpan>[
+                      TextSpan(
+                        text:
+                            '${AppLocalizations.of(context)!
+                            .welcomeToPICOS},\n',
+                        style: const TextStyle(
+                          fontWeight: FontWeight.bold,
                         ),
                       ),
-                    ),
-                    ElevatedButton(
-                      onPressed: () => _submitHandler(
-                        _loginController.text,
-                        _passwordController.text,
-                        context,
+                      TextSpan(
+                        text:
+                            AppLocalizations.of(context)!
+                            .thankYouForParticipation,
                       ),
-                      child: const Text('Submit'),
+                    ],
+                  ),
+                ),
+                const SizedBox(
+                  height: 15,
+                ),
+                SizedBox(
+                  width: 200,
+                  child: TextField(
+                    controller: _loginController,
+                    decoration: InputDecoration(
+                      border: const OutlineInputBorder(),
+                      labelText: AppLocalizations.of(context)!.login,
+                    ),
+                  ),
+                ),
+                const SizedBox(height: 10),
+                SizedBox(
+                  width: 200,
+                  child: TextField(
+                    controller: _passwordController,
+                    obscureText: true,
+                    decoration: InputDecoration(
+                      border: const OutlineInputBorder(),
+                      labelText: AppLocalizations.of(context)!.password,
+                    ),
+                  ),
+                ),
+                SizedBox(
+                  width: 200,
+                  child: PicosInkWellButton(
+                    onTap: () => _submitHandler(
+                      _loginController.text,
+                      _passwordController.text,
+                      context,
+                    ),
+                    text: AppLocalizations.of(context)!.login,
+                  ),
+                ),
+                _loginfailure
+                    ? Text(AppLocalizations.of(context)!.wrongPassword)
```

Comment:
Don't tell the user, if the password is wrong. Tell him that the credentials are wrong (more general).